implementations/ajv/main.mjs

```diff
@@ -26,8 +36,10 @@ function validateSchema(schemaPath, instancePath) {
   const validate = ajv.compile(schema);
 
   const startTime = performance.now();
-  if (!validate(instance)) {
-    process.exit(1);
+  for await (const instance of readJSONLines(instancePath)) {
```

Comment:
Note that in this loop you are reading (i.e. doing disk I/O) on the measured loop. I think to avoid any noise, we should first read every instance, create an in-memory array, and THEN benchmark

Reply:
That's a fair point. I'm not too worried about it since it should be similar across languages, but it doesn't hurt to deal with that anyway. Done.